### PR TITLE
[Feature] A new graphman command to restart a subgraph

### LINKED ISSUE #4715: [Feature] A new graphman command to restart a subgraph
### Description

This should be trivial, essentially just does `graphman pause` and `graphman resume` commands sequentially.

But restarting a subgraph is a very common thing to do, so it makes sense to have it as a separate command. Also we can add a 10-15 sec wait time between pause and resume just like we do with `rewind`

### Are you aware of any blockers that must be resolved before implementing this feature? If so, which? Link to any relevant GitHub issues.

_No response_

### Some information to help us out

- [X] Tick this box if you plan on implementing this feature yourself.
- [X] I have searched the issue tracker to make sure this issue is not a duplicate.

### PR DESCRIPTION
**Pull request by SozinM**
Closes #4715 

Kept sleep param similar to rewind command

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 - `graphman` now has two new commands `pause` and `resume` that can be used to pause and resume a deployment
 - A new table `subgraph_features` is added which tracks information like spec_version, api_version, network, features, datasources etc
 - The schema for the `subgraphFeatures` endpoint now includes data from the `subgraph_features` table
+- `graphman` now has new command `restart` that can be used to sequenteally pause and resume a deployment
```
Comment (incrypto32) [inline NEWS.md:10]: Typo for "sequentially"

File: node/src/manager/commands/assign.rs [hunk 2]
```diff
@@ -109,3 +111,15 @@ pub fn pause_or_resume(
...
 }
+
+pub fn restart(
+    primary: ConnectionPool,
+    sender: &NotificationSender,
+    search: &DeploymentSearch,
+    sleep: Duration,
+) -> Result<(), Error> {
+    pause_or_resume(primary.clone(), sender, search, true)?;
+    thread::sleep(sleep);
```
Comment (incrypto32) [inline node/src/manager/commands/assign.rs:122]: Would be great to print out a message to console.. saying something like "Waiting for x seconds"

Actually might be good to have a "Pausing <Deployment ID>"

and a "resuming <Deployment ID> too

**Review by incrypto32 — CHANGES_REQUESTED** [review]

File: node/src/manager/commands/assign.rs (hunk 2 above)
Comment (SozinM) [reply node/src/manager/commands/assign.rs:126]: Logs as of now:
```
pausing QmTGXDUdbYNuThTAUbeSA15BcQCRxX6qR1PyPN23e3JXFJ[210]
Operation completed
resuming QmTGXDUdbYNuThTAUbeSA15BcQCRxX6qR1PyPN23e3JXFJ[210]
Operation completed
```
I will add an entry for the sleeping duration.

Comment (incrypto32) [reply node/src/manager/commands/assign.rs:126]: Perfect, I'll test it out

File: NEWS.md (hunk 1 above)
Comment (SozinM) [reply NEWS.md]: Fixed

**Comment by balakhonoff** [comment]
Great job, thanks!

**Review by incrypto32 — APPROVED** [review]

**Pull request merged**